pythonforandroid/recipes/openssl/__init__.py

```diff
@@ -5,7 +5,7 @@
 
 
 class OpenSSLRecipe(Recipe):
-    version = '1.0.2f'
+    version = '1.0.2g'
```

Comment:
You are totally right, I forgot to update the patch, now the changes has been made. Many thanks to mention it !!!